src/main/kotlin/uk/gov/justice/hmpps/offenderevents/services/HMPPSDomainEventsEmitter.kt

```diff
@@ -42,8 +42,8 @@ class HMPPSDomainEventsEmitter(
   private val telemetryClient: TelemetryClient,
   private val offenderEventsProperties: OffenderEventsProperties,
 ) {
-  private val hmppsEventsTopicSnsClient: SnsAsyncClient
-  private val topicArn: String
+  private final val hmppsEventsTopicSnsClient: SnsAsyncClient
+  private final val topicArn: String
```

Comment:
Isn't this redundant? final by default...

Reply:
Just for the record, according to Intellij, it 'must be initialised, final or abstract' (a warning, but possibly an error in future)